src/main/java/org/osc/controller/nsfc/utils/RedirectionApiUtils.java

```diff
@@ -222,7 +223,28 @@ public InspectionHookEntity findInspHookByInspectedAndPort(NetworkElement inspec
             return inspectionHooks.get(0);
         });
     }
+    
+    public void addPortPairGroupToServiceFunction(List<NetworkElement> inspectionPorts,
```

Comment:
rename inspectionPorts to portPairgroups so it reads easier.

Reply:
corrected